apps/entity-service/src/entities/dto/site-report.dto.ts

```diff
@@ -102,8 +102,8 @@ export class SiteReportFullDto extends SiteReportLightDto {
       ...pickApiProperties(siteReport, SiteReportFullDto),
       lightResource: false,
       // these two are untyped and marked optional in the base model.
-      createdAt: siteReport.createdAt as Date,
-      updatedAt: siteReport.createdAt as Date,
+      createdAt: new Date(siteReport.createdAt),
+      updatedAt: new Date(siteReport.updatedAt),
```

Comment:
These really don't need the `new Date()` and I mentioned in my comment; please go back to `as Date`